services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -38,6 +39,11 @@ export const ExportedReportFieldTable = ({ section }: Props) => {
     ? twoColumnHeaderItems
     : threeColumnHeaderItems;
 
+  // The hint text is hidden for MLR page 1 (Point of Contact)
+  const reportType = report?.reportType as ReportType;
+  const hideHintText =
+    reportType === ReportType.MLR && section.form?.id === "apoc";
```

Comment:
Hey @oweingrod-coforma, I'm going to approve this for now since you're looking at adding another section to hide the hint text. I suggest though (And maybe @braxex would also have an opinion here) we move this out into the MLR.json file. For example, we could use a prop for this.

```
        "fields": [
          {
            "id": "contactName",
            "type": "text",
            "validation": "text",
            "props": {
              "label": "A. Contact name",
              "hint": "Enter the first and last name of the primary contact for this report.",
              "showHintInExport": false
            }
          },
```

That `showHintInExport` could be a passed prop so that we don't need to call out specific section, we can just reference the prop.

Reply:
That is something we could do, but it would have to be on the `form` instead of on the individual `field`. Not a bad idea though. Maybe something for @benmartin-coforma @ntsummers1 @oweingrod-coforma to collaborate on.

Reply:
I think this needs to be on a field by field basis unfortunately. There are 2 fields in the MLR figma that still have the hint text shown despite everything else not having hint text. Both of those fields are disabled, however, other disabled fields don't show their respective hint text. 

Example:
This field _doesn't_ show its hint text
```
{
  "id": "stateName",
  "type": "text",
  "validation": "text",
  "props": {
    "label": "E. State",
    "hint": "Auto-populates from your account profile.",
    "disabled": true
  }
},
```

This field _does_ show its hint text
```
{
  "id": "reportingPeriodStartDate",
  "type": "date",
  "validation": {
    "type": "date",
    "nested": true,
    "parentFieldName": "contractIncludesRemittance"
  },
  "props": {
    "label": "N. MLR reporting period start date",
    "hint": "Auto-populates from program reporting information.",
    "disabled": true
  }
},
```

And thats based off the figmas here: https://www.figma.com/file/GBktI8ZIJM9YYcdNva0ltg/MDCT---MCR?node-id=7591-162697&t=ZzRxhOyMtP2eQRp0-0

Reply:
🤦 Let's talk to design about this. Maybe we have to build it in at the form and field level so either can choose to hide it. Because show is the default, the property needs to be `hide....`. That said, I really don't like changing things like this solely for the purpose of a display-only non-interactive export view, so I'd like to push back on them and say you either get hint text or don't get hint text at the form level, not at the field level. The reporting period start date and end date being part of the entire report should be pretty obvious IMO. Start a convo with me and design please?

Reply:
@braxex We could have the property default to `showHintTextOnExport=true` and then set it to false when we don't want it to appear.

Reply:
Convo started for you!

Reply:
If we move this logic to the JSON file, we will do so under a different PR.